NYPL-discovery/discovery-front-end
# SCC-3765 - Adobe Analytics page tracking updates

**What's this do?**
This PR changes the page names emitted to Adobe Analytics, and extends the page view tracking to include some metadata derived from the URL.

I've added the utility functions adobeAnalyticsParam and adobeAnalyticsQueryString to help building those parts of the event name string.

**Why are we doing this? (w/ JIRA link if applicable)**
[Ticket](https://jira.nypl.org/browse/SCC-3765)

**Do these changes have automated tests?**
I've updated the automated test to account for the changes

**How should this be QAed?**
I've added a console.log for now to check the page name value for each page on QA.

We should also test this on the Adobe dashboard side to make sure the events are appearing as expected.
